tests/settings.py

```diff
@@ -10,9 +10,6 @@
 
 SECRET_KEY = 'secret'
 
-# TODO: Support local Resolwe server in tests
-RESOLWE_API_HOST = 'https://qa.genialis.com'
```

Comment:
I rather put it out and changed the `GenomeBrowserProcessorTestCase` to explicitly set it with `@override_settings(RESOLWE_HOST_URL='https://dummy.host.com')`